tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationOnBuilderImageTest.java

```diff
@@ -102,6 +107,40 @@ public void testCreateApplicationBasedOnBuilderImage() {
 				+ " of services: " + services.size(), services.size() == 1);
 	}
 	
+    @Test
+    public void validateJBIDE22704() {
+        new NewOpenShift3ApplicationWizard().openWizardFromExplorer();
```

Comment:
The JBIDE-22704 is about opening from Central/shell menu. Working with wizard opened from OpenShift explorer view is used on several places and its working. So there should be 2 tests. First one should use new new NewOpenShift3ApplicationWizard().openWizardFromShellMenu(); and second should use new NewOpenShift3ApplicationWizard().openWizardFromCentral();